tests/Persisters/Serializers/NativeTest.php

```diff
@@ -7,6 +7,9 @@
 use Rubix\ML\Persisters\Serializers\Native;
 use Rubix\ML\Persisters\Serializers\Serializer;
 use PHPUnit\Framework\TestCase;
+use RuntimeException;
+use stdClass;
```

Comment:
Space here

Run `$ composer check` to run the style checker, or `$composer fix` to automatically format the code per our style guide.